### PR TITLE
[Doc] Update deployment instructions to use nuget and prepacked Worker.

### PR DESCRIPTION
**Pull request by suhsteve**

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -38,34 +35,15 @@ Deploying your App on the Cloud
...
-```shell
-# For example, you can run the following on Linux using `zip`.
-foo@bar:~/path/to/app/bin/Release/netcoreapp2.1/ubuntu.16.04-x64/publish$ zip -r <your app>.zip .
-```
-3. Upload the following to a distributed file system (e.g., HDFS, WASB, ADLS, S3) that your cluster has access to:
-   * `microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar` (created in the [Build](../README.md#build) step)
+1. Follow the [Get Started](https://github.com/dotnet/spark/#get-started) guide to build and publish your app.
+
+2. Upload the following to a distributed file system (e.g., HDFS, WASB, ADLS, S3) that your cluster has access to:
+   * `microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar` (Included as part of the [Microsoft.Spark](https://www.nuget.org/packages/Microsoft.Spark/) nuget)
```
Comment (rapoth) [inline deployment/README.md]: Change Line 128 appropriately?

Comment (rapoth) [inline deployment/README.md]: Add an example along the lines of: For instance, if you want netcoreapp2.1, you'd download `xyz`

File: deployment/README.md [hunk 2]
```diff
@@ -38,34 +35,15 @@ Deploying your App on the Cloud
...
-
-2. Produce `worker.tgz` for the published files.
-```shell
-# For example, you can run the following on Linux using `tar`.
-foo@bar:~/dotnet/spark/src/csharp/Microsoft.Spark.Worker$ tar czvf worker.tgz -C bin/Release/netcoreapp2.1/ubuntu.16.04-x64/publish/ .
-```
+1. Select a [Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases) Linux netcoreapp release to be deployed on your cluster.
 
-3. Upload `worker.tgz` and [install-worker.sh](install-worker.sh) to a distributed file system (e.g., HDFS, WASB, ADLS, S3) that your cluster has access to.
+2. Upload `Microsoft.Spark.Worker.netcoreapp<dotnet_version>.linux-x64-<spark_dotnet_version>.tar.gz` and [install-worker.sh](install-worker.sh) to a distributed file system (e.g., HDFS, WASB, ADLS, S3) that your cluster has access to.
```
Comment (rapoth) [inline deployment/README.md]: Something weird in L164 sentence.

File: deployment/README.md [hunk 3]
```diff
@@ -18,13 +18,10 @@ Deploying your App on the Cloud
...
-2. Download and install [.NET Core](https://dotnet.microsoft.com/download) <span style="color: red">2.1+</span> for your operating system.
-3. Tool for creating a `tgz` file: `tar` on Linux, [7-ZIP](https://www.7-zip.org/) on Windows, etc.
-4. Tool to copy files to a distributed file system.
+1. Tool to copy files to a distributed file system.
    - ADLS, WASB &rarr; [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/)
    - S3 &rarr; [AWS CLI](https://aws.amazon.com/cli/)
-5. Download [install-worker.sh](install-worker.sh) to your local machine. This is a helper script that we will use later in the installation section to copy Spark .NET dependent files into your Spark cluster's worker nodes. install-worker.sh takes in three parameters:
+2. Download [install-worker.sh](install-worker.sh) to your local machine. This is a helper script that we will use later in the installation section to copy Spark .NET dependent files into your Spark cluster's worker nodes. install-worker.sh takes in three parameters:
    1. The Cloud Provider: `azure` or `aws`
    2. URI where `worker.tgz` is uploaded.
```
Comment (rapoth) [inline deployment/README.md]: You don't describe what `worker.tgz` is or where to obtain it from. I suggest renaming things to be more consistent across the document. Either say, download the release and rename to worker.tgz or just put the release name throughout. Whichever is easier.

Comment (rapoth) [reply deployment/README.md]: Another way, you could word it is:

URI where `worker.tgz` is uploaded (see #section for instructions on where to download this)

OR

URI where Microsoft.Spark.Worker.tgz is uploaded (see #section for instructions on where to download this) - this also indicates this is a Microsoft zip file.

Also, the release is tar.gz whereas the wording here is tgz. Can you update?

File: deployment/README.md (hunk 1 above)
Comment (rapoth) [inline deployment/README.md]: Included as part of the Nuget is fine but you should mention where they can find this jar.

Comment (rapoth) [reply deployment/README.md]: Not sure, if adding some overview of the structure at the top of the document will clear this up.

File: deployment/README.md (hunk 2 above)
Comment (suhsteve) [reply deployment/README.md]: What's wrong with L164?

File: deployment/README.md (hunk 1 above)
Comment (suhsteve) [reply deployment/README.md]: I assume you wanted this suggestion in the Microsoft.Spark.Worker section.  If so, I have just added your suggestion.

Comment (rapoth) [reply deployment/README.md]: Sorry, yes. Thank you!

File: deployment/README.md (hunk 2 above)
Comment (rapoth) [reply deployment/README.md]: Search for this text: "Configure and your Data Source and mount it" Basically, L164 got changed once you committed :) The sentence formation doesn't look right.

Comment (rapoth) [reply deployment/README.md]: I cannot point exactly at that line since it is not part of the change set.

**Pull request merged**